docs/LLS_Metallicities-Wotta/mcmc_met.py

```diff
@@ -0,0 +1,258 @@
+"""

```

Comment:
I'm not sure about this module.  Is it specific to 
your work or more generic?    Either way, if it stays
then it should go in pyigm.metallciity